main.tf

```diff
@@ -47,7 +52,7 @@ resource "google_compute_instance" "instance" {
 
   metadata = {
     ssh-keys = "${var.ssh_user}:${var.public_key}"
-    user-data = "${var.cloud_init}"
+    user-data = "${var.cloud_init} ${random_string.editor.result}"
```

Comment:
Instead of appending to `var.cloud_init` can we put a variable in there (e.g., `EDITOR_PASSWORD`) and then do a [replace](https://www.terraform.io/docs/configuration/functions/replace.html) on that?

Reply:
Sure thing!